### ISSUE TITLE
how apply "spark-cluster dynamic resource allocation"

### ISSUE THREAD

**Issue by drawhan** [big-data-europe/docker-spark#23]
how apply "spark-cluster dynamic resource allocation"

i hope apply num-executor, executor-cores and executor-memory in spark-cluster
but only executor-memory can use...

how i apply num-executors and executor-cores?

**Comment by earthquakesan** [comment]
hi @drawhan !

Can you be a bit more specific regarding what you are trying to do.

**Comment by SasidharT** [comment]
Hi ,

He is talking about the spark-default.conf where we manage the dynamic resource allocation,  like number of executors, number of cores per executor and memory allocation for each executor. even I want to know how we can manage this configuration in current docker infra you have created. I am working to create docker spark cluster with high availability for my live streaming app. @earthquakesan

**Comment by GezimSejdiu** [comment]
Hi @drawhan , 

this has been resolved via 2662a780cbed606535ff857072af843370e61413 . 

One example how to use could be: 
```
ENV SPARK_SUBMIT_ARGS "--executor-memory 16g --executor-cores 8"
```

Best regards,

**Comment by jasonkim-77** [comment]
> Hi @drawhan ,
> 
> this has been resolved via [2662a78](https://github.com/big-data-europe/docker-spark/commit/2662a780cbed606535ff857072af843370e61413) .
> 
> One example how to use could be:
> 
> ```
> ENV SPARK_SUBMIT_ARGS "--executor-memory 16g --executor-cores 8"
> ```
> 
> Best regards,

thanks
